system-contract-dapp-playground/src/utils/constants.ts

```diff
@@ -100,5 +103,22 @@ export const HEDERA_NETWORKS = {
       decimals: 18,
     },
     blockExplorerUrls: 'http://localhost:8080',
+    mirrorNodeUrl: '127.0.0.1:5600/api/v1',
   },
 };
+
+/**
+ * @notice information about protected routes
+ */
+export const PROTECTED_ROUTES = [
+  '/overview',
+  '/hts-hip-206',
+  '/hrc-719',
+  '/exchange-rate-hip-206',
+  '/prng-hip-351',
+  '/erc-20',
+  '/erc-721',
+];
+
+/** @notice hashcan baseURL */
+export const HASHSCAN_BASE_URL = 'https://hashscan.io';
```

Comment:
Maybe you can add the hashscan url to the `HEDERA_NETWORKS` object.
If not create an object here or a method in helpers to handle hashscan for each env i.e. `http://127.0.0.1:8080`, `https://hashscan.io/previewnet/dashboard`, `https://hashscan.io/testnet/dashboard` and `https://hashscan.io/mainnet/dashboard`

Reply:
Oh yeah I did add those urls to the `HEDERA_NETWORKS` they are under `blockExplorerUrl`. However, the getAccountIdFromEvmAddress() API require the `Hashscan` url to have a different param like `https://hashscan.io/testnet/accounts`. And in the future we will need different params for different purposes like `transactions` or `contracts`, etc. so I think a `HASHSCAN_BASE_URL` is necessary.